src/index.tsx

```diff
@@ -1310,6 +1310,7 @@ export default forwardRef<ActionSheetRef, ActionSheetProps>(
                               borderRadius:
                                 containerStyle?.borderRadius || undefined,
                               width: containerStyle?.width || '100%',
+                              maxWidth: containerStyle?.maxWidth || undefined,
```

Comment:
```suggestion
                              maxWidth: containerStyle?.maxWidth,
```
Setting value to `undefined` seems redundant here